src/components/InlineAlert/InlineAlert.stories.tsx

```diff
@@ -0,0 +1,141 @@
+import styled from '@emotion/styled';
+import type { Meta, StoryObj } from '@storybook/react';
+import Button from '../Button';
+import Link from '../Link';
+import { InlineAlert } from './InlineAlert';
+import { useState } from 'react';
+
+const Container = styled.div`
+  display: inline-flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 8px;
```

Comment:
for gaps we should use the spacing tokens (values provided by design)

Reply:
I will remove the gap here, since it's not required as an example.